tests/core/package.json

```diff
@@ -19,9 +20,12 @@
   "homepage": "https://github.com/WebAssembly/wasi-testsuite#readme",
   "devDependencies": {
     "@assemblyscript/wasi-shim": "^0.1.0",
+    "@types/node": "^18.11.9",
     "@typescript-eslint/eslint-plugin": "^5.42.0",
     "@typescript-eslint/parser": "^5.42.0",
     "assemblyscript": "^0.22.0",
-    "eslint": "^8.26.0"
+    "eslint": "^8.26.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.4"
```

Comment:
why do we need typescript dependency?

Reply:
It's a dependency of ts-node, where it's JIT transpiling the ts to js

Reply:
ah, I thought ts-node will pull that dependency automatically, but looks like it does not.